src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -48,7 +48,7 @@
  :<- [:wallet/balances]
  :<- [:wallet/tokens-loading?]
  (fn [[accounts balances tokens-loading?]]
-   (mapv (fn [{:keys [color address type] :as account}]
+   (mapv (fn [{:keys [color address] :as account}]
```

Comment:
Maybe we should destructure this map in another way, so that the app doesn't compile if the var doesn't exist.

```clojure
(mapv (fn [{:keys        [color address]
            account-type :type
            :as          account}]
        ,,,
        ;;line 54:
        :type (if (= account-type :watch) :watch-only :empty)
        ,,,)
      [,,,])
```

currently the app is compiling but in the line 54: `(= type :watch)` is always going to be `false`.
Just to give a little more of context on it in case it's needed:
```clojure
(= type :watch)
;; => false

type
;; => #object[cljs$core$type]
```